### PR TITLE
Support for MySQL blob fields (Read Only)

### PR DESCRIPTION
**Pull request by manuelbcd**
Support for MySQL blob fields (Read Only)

### CODE REVIEW

File: gen/controller.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 require_once __DIR__.'/../../../vendor/autoload.php';
 require_once __DIR__.'/../../../src/app.php';
-
+ta
```
Comment (blag001) [inline gen/controller.php]: mistyping no ?

**Comment by manuelbcd** [comment]
yeah sorry

**Comment by manuelbcd** [comment]
Already solved

**Comment by jonseg** [comment]
Thank you!

**Pull request merged**